lib/generators/factory_bot/model/model_generator.rb

```diff
@@ -54,7 +54,7 @@ def create_factory_file
 
       def factory_definition
         <<~RUBY
-            factory :#{singular_table_name}#{explicit_class_option} do
+            factory :#{explicit_class_name}#{explicit_class_option} do
```

Comment:
```suggestion
            factory :#{factory_name}#{explicit_class_option} do
```

I think this name captures the purpose of the method a bit better. We are using it to get the name of the factory.

Reply:
@composerinteralia Thank you for the suggestion! 😅